CHANGELOG.md

```diff
@@ -1,3 +1,91 @@
+## streamlink 0.2.0 (12-16-2016)
+
+Release 0.2.0 of Streamlink!
+
+We've done numerous changes to plugins as well as fixed quite a few which were originally failing. Amung these changes are updated docs as well as general UI/UX cleaning with console output.
```

Comment:
That's why I'm not using the UI on Github for any git related stuff, because it uses my real name I have set in my profile instead of the committer name defined in `git config --global user.name`.